nle/nethack/nethack.py

```diff
@@ -168,6 +170,8 @@ def reset(self, new_ttyrec=None, wizkit_items=None):
             _set_env_vars(self._options, self._vardir, wizkit=WIZKIT_FNAME)
         else:
             _set_env_vars(self._options, self._vardir)
+        if not self._spawn_monsters:
+            self._pynethack.set_spawn_monsters(False)
```

Comment:
I think it's a better idea to make it such that the underlying `_pynethack` object is either started with spawning on or off, without the ability to change this during its lifetime.